starter-code/src/components/FoodBox.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import "./FoodBox.css";
+
+class FoodBox extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { food: props.food };
```

Comment:
Comme tu ne vas pas changer ces valeurs ici mais seulement les lire, tu n'as pas besoin de passer par le state, tu peux utiliser les props directement.
Pour ne pas avoir à répéter `food` dans `this.props.food.name|calories|image`, tu pourrais utiliser le spread operator dans l'appel du composant